python/pyspark/sql/types.py

```diff
@@ -188,6 +199,8 @@ class IntegerType(PrimitiveType):
 
     The data type representing int values.
     """
+    def simpleString(self):
+        return 'int'
```

Comment:
is it possible to just call the JVM function? Otherwise we might be inconsistent between the two languages in the future.